### PR TITLE
Supervisor list courses

### PR DESCRIPTION
**Pull request by thaonp104**
## Related Tickets:
- [26054](https://edu-redmine.sun-asterisk.vn/issues/26054)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by thaonp104** [comment]
ready

File: app/Models/User.php [hunk 1]
```diff
@@ -85,6 +85,14 @@ public function subjectUsersActive()
...
     }
 
+    public function courses() {
+        return $this->belongsToMany(Course::class);
+    }
+
+    public function subjects() {
+        return $this->belongsToMany(Subject::class);
+    }
+
```
Comment (huyenttm-0901) [inline app/Models/User.php]: convention

File: resources/views/supervisor/manage-course/list-courses.blade.php [hunk 2]
```diff
@@ -130,20 +140,7 @@ class="table table-theme table-row v-middle dataTable no-footer"
...
-                                                    <a href="#" aria-controls="datatable" data-dt-idx="1" tabindex="0" class="page-link">
-                                                        {{--Trang--}}
-                                                    </a>
-                                                </li>
-                                                <li class="paginate_button page-item next" id="datatable_next">
-                                                    <a href="#" aria-controls="datatable" data-dt-idx="3" tabindex="0"
-                                                        class="page-link">{{ trans('supervisor.list_courses.next') }}</a>
-                                                </li>
-                                            </ul>
+                                            {{$courses->links('pagination')}}
```
Comment (huyenttm-0901) [inline resources/views/supervisor/manage-course/list-courses.blade.php]:
```suggestion
                                            {{ $courses->links('pagination') }}
```

**Comment by thaonp104** [comment]
ready

**Pull request merged**